runtime/autoload/tutor.vim

```diff
@@ -15,38 +15,28 @@ function! tutor#SetupVim()
     endif
 endfunction
 
-" Mappings: {{{1
-
-function! s:CheckMaps()
-    nmap
+" Loads metadata file, if available
+function! tutor#LoadMetadata()
+    try
+        let b:tutor_metadata = json_decode(join(readfile(expand('%').'.json'), "\n"))
+    catch
```

Comment:
@fmoralesc  with the filereadable guard this should never happen, so if it does happen it is better to get as much information as possible. Up to you, I don't want to block this PR.